index.html

```diff
@@ -9,25 +9,27 @@
   <link rel="stylesheet" href="./CSS/style.css">
 </head>
 <body>
-  <div class="container">
-    <!-- Header  -->
-    <header>
-       <nav class="top-nav">
-            <h1 class="logo">My log</h1>
-          <ul>
-              <li>
-                <img class="vector" src="./images/Img/Union.png" alt="Union">
-              </li>
-          </ul> 
-       </nav>
-       <!-- /images/Img/Union.png -->
-<!-- header inline section -->
-        <section >
-              <h2 class="greeting">
-                I'm Michael.
-                Glad to see you!
-              </h2>
-        
+  <div class="wrapper">
+     <!-- --------------------------------------header section--------------------------------->
+      <header>
+
+        <nav class="top-nav">
+                <h1 class="logo">My log</h1>
+              <ul>
+                  <li>
+                    <img class="vector" src="./images/Img/Union.png" alt="Union">
+                  </li>
+              </ul> 
+        </nav>
+
+        <section>
+            <div>
+              <div>
+                <h1 class="greeting">
+                  I'm Michael.
+                  Glad to see you!
+                </h1>
```

Comment:
- [ ] [OPTIONAL] As suggested by the previous reviewer kindly make sure you've fixed the negative space around the headline of your page as it's making your portfolio look unprofessional 👶:
    ![image](https://user-images.githubusercontent.com/104968236/229109899-2e020044-7255-4726-a370-fc3e23706f76.png)
- As I've highlighted in yellow the negative spacing isn't shown in the Figma design so I kindly suggest you fix it.
> * [ ]  [**OPTIONAL**] Kindly make sure this header section fill all the width of the screen, as it is represented in the Figma design
> 
> 
> ![Capture](https://user-images.githubusercontent.com/80458199/229096434-72519fab-5b75-4f75-b509-0fbf1ade7517.PNG)